draft-ietf-mls-protocol.md

```diff
@@ -3580,8 +3584,8 @@ message at the same time, by taking the following steps:
      `commit_secret` as the value `path_secret[n+1]` derived from the
      `path_secret[n]` value assigned to the root node.
 
-* If not populating the `path` field: Set the `path` field in the Commit to the
-  null optional.  Define `commit_secret` as the all-zero vector of length
+* If not populating the `path` field: Set the `path` field in the Commit to a
+  zero-length vector.  Define `commit_secret` as the all-zero vector of length
```

Comment:
```suggestion
* If not populating the `path` field: Set the `path` field in the Commit to the
  null optional.  Define `commit_secret` as the all-zero vector of length
```

Reverting because the `path` field has type `optional<UpdatePath>`.